### ISSUE TITLE
Question regarding versioning plan

### ISSUE THREAD

**Issue by markjm** [microsoft/TypeScript-DOM-lib-generator#1176]
Hi all -- big fan of the splitting of types out of typescript core. I looked through the already published packages and docs and dont see any versioning scheme (aside from note that it doesnt follow semver, which is fine). Is there some plan for versioning, or should we just expect some table of @types/web to typescript versions which "match"

I am wondering if it might be valuable to version `@types/web` to match the version bundled within Typescript (so `@types/web@4.5.0` would be the types included in `typescript@4.5.0`). Thoughts?

**Comment by orta** [comment]
Yep, the table is in the README: https://github.com/microsoft/TypeScript-DOM-lib-generator#typeslib-to-typescript-versions

I need to get around to merging https://github.com/microsoft/TypeScript-DOM-lib-generator/pull/1160 to add 4.5's
